addons/hw_drivers/iot_handlers/drivers/PrinterDriver_L.py

```diff
@@ -172,11 +172,25 @@ def send_status(self):
         }
         event_manager.device_changed(self)
 
-    def print_raw(self, data):
-        process = subprocess.Popen(["lp", "-d", self.device_identifier], stdin=subprocess.PIPE)
+    def print_raw(self, data, landscape=False, duplex=True):
+        """
+        Print raw data to the printer
+        :param data: The data to print
+        :param landscape: Print in landscape mode (Default: False)
+        :param duplex: Print in duplex mode (recto-verso) (Default: True)
+        """
+        options = []
+        if landscape:
+            options.extend(['-o', 'orientation-requested=4'])
+        if not duplex:
+            options.extend(['-o', 'sides=one-sided'])
```

Comment:
note: by default cups already prints in the recto/verso aka "duplex" mode so this doesn't modify the behaviour